mcp_server_snowflake/utils.py

```diff
@@ -416,37 +416,91 @@ def get_login_params() -> dict:
     --------
     >>> params = get_login_params()
     >>> params["account"]
-    ['--account', '--account-identifier', os.getenv("SNOWFLAKE_ACCOUNT")]
+    ['--account', '--account-identifier', os.getenv("SNOWFLAKE_ACCOUNT"), "Your account identifier."]
     """
     # Dict of login params supported by snowflake connector api to establish connection
-    # {Key value name : [argparse argument name, default value]}
+    # {Key value name : [argparse argument name(s), default value, help]}
+    # Each argument can have 1-2 flag values. All arguments must have a default value and help.
     login_params = {  # TODO: Add help for each argument
         "account": [
             "--account",
             "--account-identifier",
             os.getenv("SNOWFLAKE_ACCOUNT"),
+            "Your account identifier. The account identifier does not include the snowflakecomputing.com suffix.",
+        ],
+        "host": ["--host", os.getenv("SNOWFLAKE_HOST"), "Host name."],
+        "user": [
+            "--user",
+            "--username",
+            os.getenv("SNOWFLAKE_USER"),
+            "Login name for the user.",
         ],
-        "host": ["--host", os.getenv("SNOWFLAKE_HOST")],
-        "user": ["--user", "--username", os.getenv("SNOWFLAKE_USER")],
         "password": [
             "--password",
             "--pat",
             os.getenv("SNOWFLAKE_PASSWORD") or os.getenv("SNOWFLAKE_PAT"),
+            "Password for the user.",
+        ],
+        "role": [
+            "--role",
+            os.getenv("SNOWFLAKE_ROLE"),
+            "Name of the default role to use.",
```

Comment:
```suggestion
            "Name of the role to use.",
```

Similar to the note about the warehouse.

Reply:
Good call. I copied and pasted from the python connector docs. I removed the use of 'default' in the above.